internal/core/src/storage/MinioChunkManager.cpp

```diff
@@ -216,6 +216,26 @@ MinioChunkManager::BuildS3Client(
     }
 }
 
+void
+MinioChunkManager::PreCheck(const StorageConfig& config) {
+    LOG_SEGCORE_INFO_ << "start to precheck chunk manager with configuration:"
+                      << config.ToString();
+    try {
+        ListWithPrefix();
```

Comment:
List all the objects in the bucket, isn't it a bit dangerous?

Reply:
yes. I consider it,  maybe cost a little. but no find safe and little-cost solution.

Reply:
another solution, give a prefix that not exist, just test connection not real result

Reply:
should we do a simple exist fucntion for "/"

Reply:
I think it has a similar performance effect as list not exists prefix